configs/ts7100_defconfig

```diff
@@ -0,0 +1,134 @@
+CONFIG_ARM=y
+CONFIG_ARCH_MX6=y
+CONFIG_SYS_TEXT_BASE=0x87800000
+CONFIG_SPL_GPIO_SUPPORT=y
+CONFIG_SPL_LIBCOMMON_SUPPORT=y
+CONFIG_SPL_LIBGENERIC_SUPPORT=y
+CONFIG_TARGET_TS7100=y
+CONFIG_SPL_MMC_SUPPORT=y
+CONFIG_SPL_SERIAL_SUPPORT=y
+CONFIG_NR_DRAM_BANKS=1
+CONFIG_SPL=y
+CONFIG_PANIC_HANG=y
+CONFIG_SPL_RAW_IMAGE_SUPPORT=y
+CONFIG_SPL_LIBDISK_SUPPORT=y
+CONFIG_SMBIOS_PRODUCT_NAME="mx6ul_14x14_evk"
```

Comment:
```suggestion
CONFIG_SMBIOS_PRODUCT_NAME=""
```
I'd normally say we should correct it rather than remove it, but this is gone on newer u-boots and making it blank otherwise is a normal convention:
https://patchwork.ozlabs.org/project/uboot/patch/20201022141949.711984-13-sjg@chromium.org/

Reply:
It looks like that is the convention for boards that actually use/support SMBIOS. Most defconfigs omit SMBIOS entirely, so I deleted it from the 7100 (and 7250v3), the only boards of ours that had it.